workflow/5_adaptation/_adaptation.smk

```diff
@@ -133,12 +139,31 @@ rule damage_loss_timeseries_and_NPV:
     snakemake -c1 results/flood_threshold_1p0/loss_damage_npvs/waste_water_facilities_NWC_nodes_EAD_EAEL_npvs.csv
     """
     input:
+        script = "workflow/5_adaptation/damage_loss_timeseries_and_npv.py",
         network_csv = config["paths"]["network_layers"],
         growth_rates = f"{DATA}/macroeconomic_data/gdp_growth_rates.xlsx",
         summarised_damages = f"{OUTPUT}/{{protection_type}}_{{threshold}}/direct_damages_summary/{{gpkg}}_{{layer}}_EAD_EAEL.csv",
+    params:
+        baseline_year = 2019,
+        projection_end_year = 2100,
+        discounting_rate = 10
     output:
-        NPV = f"{OUTPUT}/{{protection_type}}_{{threshold}}/loss_damage_npvs/{{gpkg}}_{{layer}}_EAD_EAEL_npvs.csv"
+        NPV = f"{OUTPUT}/{{protection_type}}_{{threshold}}/loss_damage_npvs/{{gpkg}}_{{layer}}_EAD_EAEL_npvs.csv",
+        timeseries = [
+            f"{OUTPUT}/{{protection_type}}_{{threshold}}/loss_damage_timeseries/{{gpkg}}_{{layer}}_{file_suffix}.csv"
+            for file_suffix in timeseries_files
+        ],
     shell:
         """
-        touch {output.NPV}
+        python {input.script} \
+            --network-csv {input.network_csv} \
+            --growth-rates-xls {input.growth_rates} \
+            --asset-gpkg {wildcards.gpkg} \
+            --asset-layer {wildcards.layer} \
+            --protection-type {wildcards.protection_type} \
+            --threshold {wildcards.threshold} \
```

Comment:
I think these two parameters are only used to construct the output file paths, so it might be possible to replace them with a more general `output_path` parameter.